Files/Views/MainPage.xaml

```diff
@@ -192,7 +193,22 @@
         IsCompact="{x:Bind SidebarAdaptiveViewModel.IsWindowCompactSize, Mode=OneWay}"
         IsPaneOpen="{x:Bind SidebarAdaptiveViewModel.IsSidebarOpen, Mode=TwoWay}"
         SelectedSidebarItem="{x:Bind SidebarAdaptiveViewModel.SidebarSelectedItem, Mode=TwoWay}">
-        <ContentPresenter HorizontalAlignment="Stretch" Content="{x:Bind ((viewmodels:MainPageViewModel)DataContext).SelectedTabItem.Control, Mode=OneWay}" />
+        <Grid>
+            <Grid.RowDefinitions>
+                <RowDefinition Height="*" />
+                <RowDefinition Height="Auto" />
+            </Grid.RowDefinitions>
+            <ContentPresenter HorizontalAlignment="Stretch" Content="{x:Bind ((viewmodels:MainPageViewModel)DataContext).SelectedTabItem.Control, Mode=OneWay}" />
+
+            <controls:StatusBarControl
+                x:Name="StatusBarControl"
+                Grid.Row="2"
+                Grid.ColumnSpan="3"
+                x:Load="False"
```

Comment:
Instead of loading it from code behind, can you try using ` x:DeferLoadStrategy="Lazy"` for the parent grid ([documentation](https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/x-deferloadstrategy-attribute))?